### PR TITLE
fixed styling of confirm deleting viajen

### LINKED ISSUE #65: Hadia konfimasaun hamoos foto nia interface
Presiza hadia konfimasaun hamoos foto nia interface

Screenshot from 2020-04-22 22-09-48

Bele uza modal hanesan imajen ne'e:
Screenshot from 2020-04-25 10-53-04

### PR DESCRIPTION
**Pull request by marobo**
Closes #65 

hadia kofirmasaun delete viajen nia style

### CODE REVIEW

File: map/templates/map/istoriaviazen_confirm_delete.html [hunk 1]
```diff
@@ -5,9 +5,23 @@
...
-        <p>{% blocktrans %} Are you sure you want to delete {{ object }}? {% endblocktrans %}</p>
-        <button type="submit">{% trans "Confirm" %}</button>
-        <a href="{% url 'home' %}">{% trans "Cancel" %}</a> 
-    </form>
+<div class="modal-dialog">
+    <div class="modal-content delete-journey">
+        <form class="form-delete" action=" " method="POST">
+            {% csrf_token %}
+            <div class="modal-header">
+                <h4 class="modal-title">{% trans 'Are you sure?' %}</h4>
```
Comment (eraulo) [inline map/templates/map/istoriaviazen_confirm_delete.html:13]: @marobo please remove

File: map/templates/map/istoriaviazen_confirm_delete.html [hunk 2]
```diff
@@ -5,9 +5,23 @@
...
+                <a href="{% url 'home' %}"><button type="button" class="close" data-dismiss="modal" aria-hidden="true">&times;</button></a>
+            </div>
+            <div class="modal-body">
+                <p>{% blocktrans %}Do you really want to delete this "{{ object }}"?{% endblocktrans %}</p>
+                <p> {% trans 'This process cannot be undone.' %}</p>
```
Comment (eraulo) [inline map/templates/map/istoriaviazen_confirm_delete.html:18]: @marobo please remove

File: map/templates/map/istoriaviazen_confirm_delete.html [hunk 3]
```diff
@@ -5,9 +5,23 @@
...
+            </div>
+            <div class="modal-footer">
+                <a href="{% url 'home' %}"><button type="button" class="btn btn-primary cancel-delete" data-dismiss="modal">{% trans 'Cancel' %}</button></a>
```
Comment (eraulo) [inline map/templates/map/istoriaviazen_confirm_delete.html:21]:
```suggestion
                <a href="{% url 'home' %}"><button type="button" class="btn btn-default cancel-delete" data-dismiss="modal">{% trans 'Cancel' %}</button></a>
```

File: map/static/css/timor.css [hunk 4]
```diff
@@ -278,3 +278,25 @@ a:hover {
...
+
+.form-delete {
+    margin-bottom: 0px;
+}
+
+.cancel-delete {
+    border-color: var(--gray);
+    background-color: white;
+    color: var(--gray);
+
```
[the remark this answers is not in the document]
Comment (marobo) [reply map/static/css/timor.css]: Solved @eraulo

File: map/templates/map/istoriaviazen_confirm_delete.html [hunk 5]
```diff
@@ -5,9 +5,22 @@
...
+            <div class="modal-header">
+                <h4 class="modal-title">{% trans 'Confirmation delete' %}</h4>
+                <a href="{% url 'home' %}"><button type="button" class="close" data-dismiss="modal" aria-hidden="true">&times;</button></a>
+            </div>
+            <div class="modal-body">
+                <p>{% blocktrans %} Are you sure you want to delete "{{ object }}"? {% endblocktrans %}</p>
+            </div>
+            <div class="modal-footer">
+                <a href="{% url 'home' %}"><button type="button" class="btn btn-default cancel-delete" data-dismiss="modal">{% trans 'Cancel' %}</button></a>
+                <button class="btn btn-danger btn-ok">{% trans 'Delete' %}</button>
```
Comment (eraulo) [inline map/templates/map/istoriaviazen_confirm_delete.html:21]:
```suggestion
                <button class="btn btn-danger ">{% trans 'Delete' %}</button>
```

**Pull request merged**